src/lib/transition_router/transition_router.ml

```diff
@@ -66,32 +66,88 @@ module Make (Inputs : Inputs_intf) :
     recieved_time |> Time.to_span_since_epoch |> Time.Span.to_ms
     |> Unix_timestamp.of_int64
 
+  let create_bufferred_pipe () =
+    Strict_pipe.create (Buffered (`Capacity 10, `Overflow Drop_head))
+
+  type tfc_transition =
```

Comment:
don't use bespoke acronyms please (let's not let `tfc` get into the codebase anywhere)